k8s/charts/openebs/values.yaml

```diff
@@ -58,42 +72,50 @@ ndm:
   image: "quay.io/openebs/node-disk-manager-amd64"
   imageTag: "v0.3.5"
   sparse:
-    enabled: "true"
     path: "/var/openebs/sparse"
     size: "10737418240"
     count: "1"
   filters:
     excludeVendors: "CLOUDBYT,OpenEBS"
-    excludePaths: "loop,/dev/fd0,/dev/sr0,/dev/ram,/dev/dm-,/dev/md"
+    excludePaths: "loop,fd0,sr0,/dev/ram,/dev/dm-,/dev/md"
```

Comment:
But in operator YAML, it is /dev/fd0,/dev/sr0. Just curious, without dev path, will these 2 items can NDM detect and exclude? cc @akhilerm

Reply:
yes @ranjithwingrider . Without complete `devPath` also NDM can detect and exclude.